lib/pharos/etcd/client.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+require 'json'
+
+module Pharos
+  module Etcd
+    class Client
+      CURL = 'sudo curl -s --connect-timeout 2 --cacert /etc/pharos/pki/ca.pem --cert /etc/pharos/pki/etcd/client.pem --key /etc/pharos/pki/etcd/client-key.pem https://localhost:2379'
+
+      def initialize(ssh)
+        @ssh = ssh
+      end
+
+      def healthy?
+        response = @ssh.exec!("#{CURL}/health")
+        response.stdout == '{"health": "true"}'
+      rescue
+        false
+      end
+
+      # @return [Array<Hash>]
+      def members
+        result = @ssh.exec("#{CURL}/v2/members")
+        JSON.parse(result.stdout)['members']
+      rescue JSON::ParserError
+        []
```

Comment:
Bad idea, just let it raise instead? Is there some known case where this happens?